README.md

```diff
@@ -1 +1,144 @@
-# DELETE ME
+# ACM SIGDOC Structured Authoring article
+
+Articles that ACM SIGDOC Stuructured Authoring committee members write are hosted on this repository. Reviews take place by using the branching system, which separate articles into different branches during their review lifecycle. The final version of the articles and other resources are published on our committee resources website:
+
+- [ACM SIGDOC Structured Authoring Committee Resources](https://acm-sigdoc-structured.org/index.html)
+
+# How to contribute
+
+Before you contribute, reach out to us at authors@acm-sigdoc-structured.org. After onboarding, you can pick from a list of topics and begin writing your article.
+
+# General workflow
+
+We support articles written in Markdown or Microsoft Word. Regardless of what format you choose, the workflow process is broken down into five steps that are represented by branches respectively:
+
+- `initial_draft`
+- `technical_review`
+- `instructional_review`
+- `editorial_review`
+- `final_draft`
```

Comment:
We would need to change our main branch to `final_draft` and make sure we remove any content from there that isn't ready to publish.